src/main/java/core/basesyntax/Triangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Triangle extends Figure implements Drawable, AreaCalculator {
```

Comment:
```suggestion
public class Triangle extends Figure {
```
that's redundant since Figure already implements these interfaces